api/helpers.go

```diff
@@ -29,6 +30,14 @@ func getToken(ctx context.Context) *jwt.Token {
 	return obj.(*jwt.Token)
 }
 
+func getConfig(ctx context.Context) *conf.Configuration {
```

Comment:
Nice - I'll aim to get that integrated when I move a few more things to the context. Will also want to do something around the db, so all db calls pass through some abstraction (will also make testing easier, just don't want to spend a lot of time on tests while the design is still changing fairly frequently).